src/main/scala/devices/tilelink/Zero.scala

```diff
@@ -19,7 +19,8 @@ class TLZero(address: AddressSet, beatBytes: Int = 4)(implicit p: Parameters)
       region = RegionType.UNCACHED,
       executable = true,
       mayDenyGet = false,
-      mayDenyPut = false),
+      mayDenyPut = false,
+      minLatency = 1),
```

Comment:
That is a lie. The implementation clearly has minLatency=0.

Reply:
then it has always been lying.

Reply:
previously DevNull device always reported that it had minLatency=1. So if this is actually 0 then it should report as 0.

Reply:
This is the line I am referring to: https://github.com/chipsalliance/rocket-chip/blob/b0ed2be996f3222225f23fb9390ab75df6c57607/src/main/scala/devices/tilelink/DevNull.scala#L55